valkey.conf

```diff
@@ -687,6 +687,12 @@ repl-diskless-load disabled
 # memory to accommodate the buffer during synchronization. However, this tradeoff is 
 # generally beneficial as it prevents potential performance degradation on the primary 
 # server, which is typically handling more critical operations.
+#
+# Additionally, it's important to note that when toggling this configuration on or off 
```

Comment:
I wouldn't say "important to note". It's kind of expected behaviour and it's safe to toggle the config.

```suggestion
# When toggling this configuration on or off 
```